corehq/apps/accounting/tasks.py

```diff
@@ -227,6 +235,19 @@ def send_subscription_reminder_emails(num_days, exclude_trials=True):
             subscription.send_ending_reminder_email()
 
 
+def send_subscription_reminder_emails_dimagi_contact(num_days):
+    today = datetime.date.today()
+    date_in_n_days = today + datetime.timedelta(days=num_days)
+    ending_subscriptions = (Subscription.objects
+                            .filter(is_active=True)
+                            .filter(date_end=date_in_n_days)
```

Comment:
Interesting, thanks for the clarification! I totally misinterpreted how celery was working in the backend. I think for now I'll leave it, since this is how all the other accounting reminders are scheduled. If we want to go through and change how these are scheduled, I can make a ticket and put it on my backlog, since I think adding it here might be out of scope.